packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/nodejs/integrationsteps.md

```diff
@@ -1,64 +1,97 @@
+Display all of the authenticators that the user has enrolled and are ready for use. You can find the names and IDs of the available authenticators in the `options` array.
 
 <div class="half border">
 
-![Displays a Select Authenticator page that includes an email field and a Select button.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-screen-verify-nodejs.png)
+![A Choose Your Authenticator page with the choices of email and phone, and a Next button](/img/oie-embedded-sdk/wireframes/choose-authenticator-email-phone-form-g2r28.png)
 
 </div>
 
-When the user selects the **email** factor, call `idx.authenticate` and pass in the authentication email authentication key, `({ authenticator: AuthenticatorKey.OKTA_EMAIL })`. With this call, Okta sends a verification code to the user's email.
+### 4: The user submits the email authenticator
 
-If the call is successful, the method returns a status of `Idx.Status:PENDING`, which indicates that the SDK is ready for the verification code. The `nextStep` field identifies the input parameter of the next step, which is a verification code in this case:
+When the user submits the email authenticator, pass the authenticator's ID as a parameter to `OktaAuth.idx.authenticate()`.
 
-```JavaScript
-status, // IdxStatus.PENDING
-  nextStep: {
-    inputs // [{ name: 'verificationCode', ... }]
-}
+```javascript
+const transaction = await authClient.idx.authenticate({ authenticatorId });
+handleTransaction({ req, res, next, authClient, transaction });
 ```
 
-The next step is to redirect the user to the email verification code page.
-
-### 3: Show the email verification code page
+### 5: The user verifies their identity with the email authenticator
 
-If not already done, build the email verification code page that accepts the code from the email.
+Identity Engine sends a verification email to the user if the call is successful. The returned `IdxTransaction` object has a status of `PENDING` and `nextStep.name` of `challenge-authenticator`. This status indicates that Identity Engine is waiting for the user to check their email and either click the magic link or enter the OTP.
 
-<div class="half border">
+To learn how to support verification with magic links or OTP, see the [Okta email integration guide](/docs/guides/authenticators-okta-email/nodeexpress/main/#_3-display-otp-input-page).
 
-![Displays a Challenge Email Authenticator page that includes a verification code field and a Verify button.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-verify-email-code-nodejs.png)
+### 6: Your app handles an authentication success response
 
-</div>
+When the user correctly verifies their identity using the email authenticator, the returned `IdxTransaction` object has a status of `SUCCESS`, along with ID and access tokens. The user has now signed in.
```

Comment:
```suggestion
When the user correctly verifies their identity using the email authenticator, the returned `IdxTransaction` object has a `SUCCESS` status, along with ID and access tokens. The user has now signed in.
```

Reply:
Not changing. This is how it is phrased int eh other guides.